examples/axp_optimizer_example.py

```diff
@@ -0,0 +1,134 @@
+from fds.analyticsapi.engines.model.optimizer_output_types import OptimizerOutputTypes
+from fds.analyticsapi.engines.model.optimizer_trades_list import OptimizerTradesList
+from fds.analyticsapi.engines.model.optimization import Optimization
+
+from urllib3 import Retry
+
+host = "https://api.factset.com"
+username = os.environ["ANALYTICS_API_USERNAME_SERIAL"]
+password = os.environ["ANALYTICS_API_PASSWORD"]
+
+
+def main():
+    config = Configuration()
+    config.host = host
+    config.username = username
+    config.password = password
+    # add proxy and/or disable ssl verification according to your development environment
+    # config.proxy = "<proxyUrl>"
+    config.verify_ssl = False
+
+    # Setting configuration to retry api calls on http status codes of 429 and 503.
+    config.retries = Retry(total=3, status=3, status_forcelist=frozenset([429, 503]), backoff_factor=2,
+                           raise_on_status=False)
+
+    api_client = ApiClient(config)
+
+    try:
+        # {
+        #     "data": {
+        #         "strategy": {
+        #             "id": "Client:/Optimizer/CN_TEST"
+        #         },
+        #         "account": {
+        #             "id": "CLIENT:/OPTIMIZER/IBM.ACCT"
+        #         },
+        #         "optimization": {
+        #             "riskmodeldate": "09/01/2020",
+        #             "backtestdate": "09/01/2020",
+        #             "cashflow": "0"
+        #         },
+        #         "outputtypes": {
+        #             "trades": {
+        #                 "identifiertype": "SedolChk",
+        #                 "includecash": false
+        #             }
+        #         }
+        #     }
+        # }
+        axp_optimizer_strategy = AxiomaEquityOptimizerStrategy(id="Client:/Optimizer/CN_TEST")
+        axp_optimizer_account = OptimizerAccount(id="CLIENT:/OPTIMIZER/IBM.ACCT")
+        axp_optimizer_optimization = Optimization(
+            risk_model_date="09/01/2020",
+            backtest_date="09/01/2020",
+            cashflow="0"
+        )
+        axp_optimizer_trades_list = OptimizerTradesList(identifier_type="SedolChk", include_cash=False)
+        axp_optimizer_output_types = OptimizerOutputTypes(trades=axp_optimizer_trades_list)
+        axp_optimizer_parameters = AxiomaEquityOptimizationParameters(
+            strategy=axp_optimizer_strategy,
+            output_types=axp_optimizer_output_types,
+            account=axp_optimizer_account,
+            optimization=axp_optimizer_optimization
+        )
+        axp_optimization_parameters_root = AxiomaEquityOptimizationParametersRoot(data=axp_optimizer_parameters)
+
+        axp_optimizations_api = AXPOptimizerApi(api_client)
+
+        post_and_optimize_response = axp_optimizations_api.post_and_optimize(
+            axioma_equity_optimization_parameters_root=axp_optimization_parameters_root,
+            _return_http_data_only=False
```

Comment:
Is `_return_http_data_only` false by default?

Reply:
No, it's true by default

Reply:
@saigiridhar21 is it worth make this change in open api generator?

Reply:
yes. Considering that we return multiple status codes, it is better to make the change.